### ISSUE TITLE
can not get ocp version

### ISSUE THREAD

**Issue by SubChange** [bernhard-42/jupyter-cadquery#82]
I have installed package cadquery and jupyter-cadquery through following commands on `win10` successfully:

```text
conda install -c cadquery cadquery
conda install -c cadquery ocp
```
however, when I running the given examples in `example/1-cadquery.ipynb`, An error occurred

![image](https://user-images.githubusercontent.com/28522692/175914670-80840228-ff37-4f58-9633-dfefe2f8d1c1.png)

It seems that I have no file in the corresponding path `f"{os.environ['CONDA_PREFIX']}/lib/libTKBRep.*.*.*"` - function `ocp_version` in `ocp_utils.py` , what I have in folder `envs/env_name/Lib/site-packages` are as follows:

![image](https://user-images.githubusercontent.com/28522692/175915784-364e351e-d390-4fd8-b94a-ae15abb7b10c.png)

Is there any updates for this function?

**Comment by bernhard-42** [comment]
libTK... libraries are not from OCP but from occt and occt does not install into site-packages, but into '$CONDA_prefix/lib'. Is occt properly installed for you?

**Comment by bernhard-42** [comment]
Not sure what happened here, but to be on the safe side, 5e latest version 3.1 catches this exception

**Comment by bernhard-42** [comment]
fixed in 3.1. Will close. If you still find issues, please reopen
